crates/nu-cli/tests/completions/mod.rs

```diff
@@ -363,6 +400,146 @@ fn file_completions() {
     match_suggestions(&expected_paths, &suggestions);
 }
 
+#[derive(Clone)]
+pub struct CustomCommandWithAnyArgs;
+
+impl Command for CustomCommandWithAnyArgs {
+    fn name(&self) -> &str {
+        "list"
+    }
+
+    fn description(&self) -> &str {
+        "Mock def command."
+    }
+
+    fn signature(&self) -> Signature {
+        Signature::build("list").rest("args", SyntaxShape::Any, "rest of the arguments")
+    }
+
+    fn run(
+        &self,
+        _engine_state: &EngineState,
+        _stack: &mut Stack,
+        _call: &Call,
+        _input: PipelineData,
+    ) -> Result<PipelineData, ShellError> {
+        todo!()
+    }
+}
```

Comment:
Rather than defining this in Rust, I think you could do the following to make it shorter.
```
let command = r#"def list [ ...args: any ] {}"#;
assert!(support::merge_input(command.as_bytes(), &mut engine, &mut stack).is_ok());
```
There are other places in this file doing that, see https://github.com/nushell/nushell/blob/88d27fd607cabf860843981532921338264c7e30/crates/nu-cli/tests/completions/mod.rs#L46 as an example.

Reply:
True, I did look around for this but did not spot that. I think the `assert!` distracted me to think it were a line without side effects :D

Reply:
Ah, yeah, it's easy to miss that. Perhaps a `.expect`/`.unwrap` would be more obvious. It would also show the contents of the `Err` variant, I think? But that's outside the scope of this PR, I guess. Maybe a later PR can look into `assert!(foo().is_ok())` vs `foo().expect("asdf")`.